File: PHP/car.php
<?php

class Car {
  public $id;
  public $licence;
  public $driver;
  public $passenger;

  public function function___construct($licence, $driver) {
    $this-> licence = $licence;
    $this-> driver= $driver;
  }

  public function PrintDataCar(){
    echo "licence:$this->licence,conductor:{$this->driver->name},document:{$this->driver->document}";
  }
}
?>

File: Java/Car.java
package Java;

class Car {
  private Integer id;
  private String license;
  private Account driver;
  protected Integer passenger;

  public Car(String license, Account driver) {
    this.license = license;
    this.driver = driver;
  }

  void printDataCar() {
    if(passenger != null){
      System.out.println("License: " + license + " Driver: " + driver.name + "Passenger: " + passenger);
    }
  }

  public Integer getPassenger(){
    return passenger;
  }
  
  public void setPassenger(Integer passenger) {
    if (passenger == 4){
    this.passenger = passenger;
    }else{
      System.out.println("Nesesitas asignar 4 pasjeros");
    }
  }

}